.gitmodules

```diff
@@ -5,3 +5,4 @@
 [submodule "test/tests"]
 	path = test/jsontests
 	url = https://github.com/ethereum/tests.git
+  branch = invalid-ops
```

Comment:
Once this is in, we will need to remove it.  Other people will start changing this line, for no benefits.